spec/exasolrls/RlsAdapter_spec.lua

```diff
@@ -5,7 +5,13 @@ local mockagne = require("mockagne")
 local adapter_capabilities = require("exasolrls.adapter_capabilities")
 local RlsAdapter = require("exasolrls.RlsAdapter")
 require("exasolrls.RlsAdapterProperties")
-local pom = require("spec.pom.reader")
+local Reader = require("spec.pom.Reader")
```

Comment:
```suggestion
local PomReader = require("spec.pom.Reader")
```
❓

Reply:
Changed from `pom.Reader` to `PomReader`